LarryKarani/iReporterApi
# #162327515 test user auth

#### What does this PR do?
This PR tests the  user authentication functionality
#### Description of tasks to be completed?
- create tests for login 
- create tests for signup
### How should this be manually tested
- clone the repo
- create a virtual environment by running `source env\bin\actiavate`
- install the project's dependencies by running ``pip install -r requirements.txt
- run `$ pytest ` to see if the tests are passing
### Relevant PT stories
#162327515
